core/src/main/java/com/novoda/noplayer/NoPlayerView.java

```diff
@@ -86,7 +93,24 @@ public void removeControls() {
     private final Player.VideoSizeChangedListener videoSizeChangedListener = new Player.VideoSizeChangedListener() {
         @Override
         public void onVideoSizeChanged(int width, int height, int unappliedRotationDegrees, float pixelWidthHeightRatio) {
-            aspectRatioChangeListener.onVideoSizeChanged(width, height, unappliedRotationDegrees, pixelWidthHeightRatio);
+            aspectRatioChangeListener.forwardVideoSizeChanged(width, height, unappliedRotationDegrees, pixelWidthHeightRatio);
+        }
+    };
+
+    private final Player.StateChangedListener stateChangedListener = new Player.StateChangedListener() {
+        @Override
+        public void onVideoPlaying() {
+            shutterView.setVisibility(INVISIBLE);
```

Comment:
We opted to use the `SimpleExoPlayerView` for both `MediaPlayer` and `ExoPlayer` but we found that the `shutterView` is rendered on top of the `surfaceView` to prevent artifacts appearing during the prepare phase of setting up the video. Due to this being used on `MediaPlayer` we never reach the `onFirstFrameRendered` call which would have hidden the shutter, so now we grab the shutter ourselves and hide it when the playback starts.